app/src/main/java/com/shagalalab/constitution/data/dao/PartDao.kt

```diff
@@ -8,5 +9,8 @@ import com.shagalalab.constitution.data.models.PartModel
 interface PartDao {
 
     @Query("SELECT * FROM parts")
-    fun getParts(): List<PartModel>
+    fun getParts(): LiveData<List<PartModel>>
+
+    @Query("SELECT * FROM parts WHERE lang_id=:langId")
+    fun getPartListByLangId(langId: Int): MutableList<PartModel>
```

Comment:
better to have `getPartsByLanguage()`. Also I think I explained the difference between `List` and `MutableList` in one of the previous [PRs](https://github.com/shagalalab/constitution-android/pull/6#discussion_r326888408). So basic rule is, `List` should be used wherever possible. You can use `MutableList`, but it should be only used for internal usage, no one from outside (class/method) should have access to it.